digdag-server/src/main/java/io/digdag/server/rs/ProjectResource.java

```diff
@@ -428,12 +431,22 @@ public RestSessionCollection getSessions(
 
             List<StoredSessionWithLastAttempt> sessions;
             if (workflowName != null) {
-                sessions = ss.getSessionsOfWorkflowByName(proj.getId(), workflowName, validPageSize, Optional.fromNullable(lastId));
+                sessions = ss.getSessionsOfWorkflowByName(proj.getId(), workflowName, Optional.fromNullable(lastId));
```

Comment:
When many records returned, it will consume too much memory.

Reply:
@yoyama 
To avoid consuming too much memory, i will fix to select limited records  by using `limit` and `offset` in query.
In order to realize the above, it is necessary to add the page_number to the argument.
Should I add a new method without changing the interface of the existing method?